canvas_account_admin_tools/settings/local.py

```diff
@@ -21,11 +21,34 @@
 ICOMMONS_REST_API_SKIP_CERT_VERIFICATION = True
 
 SELENIUM_CONFIG = {
+   'account_admin': {
+      'relative_url': 'accounts/1/external_tools/20',
+   },
+   'canvas_base_url': CANVAS_URL,
+   'course_info_tool': {
+      'relative_url': 'accounts/1/external_tools/20',
+      'test_course': {
+         'cid': '339331',
+         'term': 'Spring',
+         'title': 'Latin Paleography and Manuscript Culture: Seminar',
+         'school': 'Divinity School',
+         'type': 'Only courses without sites',
+         'year': '2014',
+      },
+      'test_users': {
+         'existing': {
+            'user_id': '20299916'
+         },
+         'new': {
+            'role': 'Teacher',
+            'user_id': '30833767'
+         }
+      }
+   },
+   'run_locally': True,
```

Comment:
This probably wants to be `SECURE_SETTINGS.get('selenium_run_locally', False)`, to match what we're doing in other projects.